src/test/ui/parser/intersection-patterns.stderr

```diff
@@ -0,0 +1,33 @@
+error: pattern on wrong side of `@`
+  --> $DIR/intersection-patterns.rs:13:9
+   |
+LL |         Some(x) @ y => {}
+   |         -------^^^-
+   |         |         |
+   |         |         binding on the right, should be to the left
+   |         pattern on the left, should be to the right
+   |         help: switch the order: `y@Some(x)`
```

Comment:
We surely want spaces here:
```rust
y @ Some(x)
```
I.e. we should change `pprust:: pat_to_string`.

Reply:
Make an issue? :)

Reply:
...or I can fix it here. =P